components/tarotSpreadSelectionBox/TarotSpreadSelectionBox.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import { Meta, StoryFn } from '@storybook/react';
+import TarotSpreadSelectionBox, {
+  TarotSpreadSelectionProps
+} from './TarotSpreadSelectionBox';
+
+export default {
+  title: 'Components/TarotSpreadSelectionBox',
+  component: TarotSpreadSelectionBox,
+  argTypes: {
+    amountOfCards: {
+      control: {
+        type: 'select'
+      },
+      type: 'number',
+      options: [1, 3, 5, 7]
+    },
+    title: { control: 'text' },
+    description: { control: 'text' }
+  }
+} as Meta<typeof TarotSpreadSelectionBox>;
+
+const Template: StoryFn<typeof TarotSpreadSelectionBox> = (
+  args: TarotSpreadSelectionProps
+) => <TarotSpreadSelectionBox {...args} />;
+
+export const Default = Template.bind({});
```

Comment:
Maybe we can add a one or two more showing the options with the most cards since we had trouble with it spilling out over the container at one point

Reply:
Sure, I'll work on adding these. After looking this over myself, I might go ahead and also pull the parent container(s) into `tarotSpread` to allow it to completely stand on it's own as a component. That way it'll be much easier to re-use later.

Reply:
These have been added to Storybook.

I did also end up moving the parent containers into `tarotSpread` and adding the appropriate props. This should effectively make it a fully reusable component.

Additionally, I added `index.ts` files to the components for simpler imports and to more closely follow the existing components.

This is working without errors on my machine, but I encourage you to test it out before merging since I had to rebuild `node_modules` along the way.

Cheers!